peer/brontide.go

```diff
@@ -1775,7 +1807,7 @@ func messageSummary(msg lnwire.Message) string {
 			msg.ChanID, msg.ID, msg.FailureCode)
 
 	case *lnwire.Warning:
-		return fmt.Sprintf("%v", msg.Error.Error())
+		return fmt.Sprintf("%v", msg.Warning())
```

Comment:
If we keep the existing `Error` method, then we can pass it around as a normal `error`. Not sure if we 100% want that behavior though...

Reply:
I would leave it like this so we do not have any error in the future using the `Error` interface for warnings.